ecl/hql/hqlattr.cpp

```diff
@@ -3868,6 +3868,29 @@ double queryLikelihood(IHqlExpression * expr)
     return likelihoodExpr->queryValue()->getRealValue();
 }
 
+double queryActivityLikelihood(IHqlExpression * expr)
+{
+    double filterLikelihood = 1.0;
+    unsigned childCount = expr->numChildren();
```

Comment:
minor: You can use the following macro to iterate through children:

ForEachChildFrom(i, expr, 1)